qemu/tests/cfg/multi_disk.cfg

```diff
@@ -93,7 +93,10 @@
                     stg_params += "drive_port:range(0,16383,63) "
                 - multi_scsiid_lun:
                     only Linux
-                    stg_params += "drive_unit:range(0,255,1,3) "
+                    !s390x:
+                        stg_params += "drive_unit:range(0,255,1,3) "
+                    s390x:
+                        stg_params += "drive_unit:range(0,8,1,3) "
```

Comment:
Is there any technics background to set the max scsi-id of s390x to be less than 8 ?  The max scsi-id on s390x is still 255 now.

Reply:
This limit came from `virttest.qemu_devices.qcontainer` where in case of `ccw` the ranges were set according to `lsi` device. Anyway I just verified it works well with 255 range so moving this fix from here to https://github.com/avocado-framework/avocado-vt/pull/1200